cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -12,3 +12,14 @@ args:
 
 imagePullSecrets:
 - dockerhub
+
+clusterStacks:
+  aws:
+    version: v0.1.0
+    deploy: false
+  gcp:
+    version: v0.1.0
+    deploy: false
+  azure:
+    version: v0.1.0
```

Comment:
@jbw976 are these versions going to bump to v0.2.0 in conjunction with Crossplane v0.4.0?

Reply:
yes, these should get bumped to v0.2.0 along with Crossplane v0.4.0.

Reply:
note however that these are just the default values, and that whoever is invoking Helm can put whatever values they want in.  We could leave this as is and GitLab when they install the Crossplane helm chart can specify v0.2.0 instead of this default.

I think we'll change the default to v0.2.0 when those ship anyways, but just wanted to be clear these are only the defaults.

Reply:
that's super helpful, thanks @jbw976 ... also think that will unblock @turkenh since they can start passing `master` in the value instead of relying on the default version.